### ISSUE TITLE
Hi, I'm Shirley and I can't wait to work with all of you this semester :))

### ISSUE THREAD

**Issue by ShirleyZ11** [TheBreakingGoodProject/Dalyell-Projects#4]
I'm currently studying a B Sci/ B Adv Studies, majoring in Applied Medical Science and Pharmacology. I am excited for the project because I think it will be interesting to understand and become more aware of the challenges people face everyday when it comes to accessing medication. More importantly, I think this is a topical issue that should be addressed so that accessibility to essential medicines becomes easier for all. 

Common, everyday medicines are frequently misused due to misconceptions about the science community or poor knowledge, so it is important that the public become more informed of the essential medicines available to us. I hope to collaborate with everyone and design a valuable end product that aims to educate others but also give us more insight into the difficulties surrounding access to medication or the understanding of these medicines. 

Keen to meet everyone and work together!

**Comment by yaelago** [comment]
Hi Shirley, welcome!
Looking forward to hearing about your research journey and learning.

**Comment by ShirleyZ11** [comment]
21/3/21

I have completed Challenge 1 from E$$ENTIAL MEDICINE$ and so far I am finding the project very enjoyable. I find it interesting to be able to learn more about something that I am passionate about, which is medicine and pharmacology. For the challenge, we were asked to identify any current news on topics related to medicines, including recent price hikes, side effects of drugs or new emerging drugs. In my research, I had the opportunity to look closer into articles surrounding a new COVID-19 vaccine, AstraZeneca, by Oxford University, which saw a very rare but serious side effect of blood clots, especially prevalent in younger women aged 55 and under. Currently, more monitoring is being conducted on this vaccine and efforts are made to determine whether there is a direct causal link between the vaccine and blood clots. Through this task, I was able to develop my own research skills and look critically at articles. It also highlighted to me, the importance of research within the pharmacology and medicine field- to continuously improve existing drugs or discover new and more efficient drugs to improve the quality of life for people. Overall, I think this challenge was a good introduction into E$$ENTIAL MEDICINE$ and the aim of our final project. I am excited to continue working through the challenges and gain a greater understanding of this topical issue which affects people globally.

**Comment by ShirleyZ11** [comment]
28/3/21

After completing the whole set of challenges for my five chosen medicines, I have gained a lot of insight into their different uses and see the significance in making each of these essential medicines more accessible to all. 

1. Insulin
The first medicine I researched was insulin. Throughout the world, more people are becoming diabetic due to a poor diet or sedentary lifestyle. However, the accessibility to insulin has certainly not increased with this trend. Prices of insulin are reaching $700 to $2000 a month for Type 2 diabetics and these price hikes are especially noticeable in the US. Thankfully, as with many other essential medicines in Australia, they fall under the Pharmaceutical Benefits Scheme (PBS) so only cost around $30 each. Eli Lilly and Co was the first company to produce and own insulin, under the brand name Humulin. 

I found that researching information on insulin did not present too big a challenge as it is such a common medicine that is used everyday by some. I was able to use the instructions and tips on the project page to help me find reliable sites. In my research I also realised that since insulin was first produced in the 1980s, there have been many other companies also producing versions of their own insulin, and that it now comes in many forms. This is to accommodate to the different needs of diabetic patients, whether they need long term, short term or medium term dosage of insulin.

2. Fluoxetin
Fluoxetin, commonly sold under the brand name Prozac, is an antidepressant used to treat depression, bulimia or compulsive disorders. I was also lucky to come across a lot of information related to this medicine due to its recent 'popularity' as a drug that could significantly inhibit replication of COVID-19 and decrease viral protein expression. This research was being conducted by a team of scientists at Julius Maximilian University of Würzburg in Germany and there studies show promising results for fluoxetin to play an important role in the treatment of COVID-19 affected patients. Unfortunately, like other drugs in the US, their prices have increased dramatically from $9 to $69, over the span of just a few years, and most companies say that production costs are responsible for this hike. 

I am interested to learn more about the different medicines which have been repurposed to treat illnesses. I found it especially interesting that drugs like fluoxetin, which are more older in their discovery, can still remain relevant today, even with modern technology and more advanced research. 

3. Digoxin
Although I was fortunate enough to find a lot of information for the previous two medicines, digoxin proved to be a bigger challenge. The medicine was first discovered in 1785 from the purple foxglove plant but was widely available in 1982. Though it was commonly used to control heart problems, including arrhythmias and atrial fibrillation, studies have shown that in some cases, digoxin actually increases mortality in patients with AF. Due to this, digoxin is now less commonly used and I think this could have attributed to the limited and sparse information on the drug. 

I also found it difficult to find information on the amount that drugs have increased. Despite the many sources which mention of current price of medicines like digoxin, not many compare the prices and show the amount of increase over time. The process of lengthy research taught me many skills to efficiently find information, and what key words I could use to simplify my search. These skills, I used for my last two rounds of the challenge and it proved very useful.

4. Dexamethasone 
This was another medicine that recently became more popular for its potential as a treatment for COVID-19. Though the drug was first sold in 1982 to treat rheumatic problems, skin diseases and allergies due to its anti-inflammatory and immunosuppressant effects, researchers from the University of Nottingham UK have studied its positive effects on COVID patients. A paper by the United Kingdom National Health Services (NHS) saw the value of dexamethasone, saying that: _“Dexamethasone reduced deaths by one-third in ventilated patients and by one fifth in other patients receiving oxygen only.”_

However, dexamethasone prices have increased by 137% in the past year, primarily due to its potential effectiveness to treat COVID. Although it is still not known if the medicine does truly have this capability, Donald Trump had used it during his recovery from COVID which sparked widespread attention. I realised that much like other personalities in the news, the influence a world known person is significant and the general public is prone to believing them. Shortly after the announcement of Trump's recovery, the US suffered a shortage of dexamethasone, forcing prices to go up. 

My research into this drug made me wonder if other anti-inflammatory drugs and immunosuppressants also have the same potential to treat COVID. I also enjoyed reading about all the different articles and their opinions on the recent news- it was interesting to see what different stakeholders think about the matter. 

5. Levonorgestrel 
The final medicine I chose to learn more about was levonorgestrel. By this time, I was already feeling a lot more comfortable with the research process. My journey to finding recent news, price hikes and the origins of the different medicines taught me valuable skills for effective research. I became more familiar with looking at research articles and filtering out unreliable sites. It was also interesting to learn about the mechanisms of action and uses for different drugs, being a pharmacology student. 

Levonorgestrel is a hormone medication that can be used as a contraceptive, commonly known as the morning-after pill. Like many other medicines, multiple companies now supply different forms of levonorgestrel and this creates a competitive market leading to the gradual price hike. Unlike some other medicines, however, levonorgestrel has not seen too significant of a price hike due to its importance as a contraceptive. There has also been recent studies that show levonorgestrel intrauterine devices to induce a response to precancer. This presents a lot of potential and could be important in the future for endometrial cancer.

**Comment by ShirleyZ11** [comment]
Through completing the tasks from the E$$ENTIAL MEDICINE$ challenges, I saw the significance of thorough research and how despite the accessibility of the internet, access to accurate information can often be difficult. I especially found it challenging to find reliable information on the price hikes of different drugs and there was a lot of varying statististics which meant filtering the different sources was important. The steps in the project helped guide me to find trustworthy websites and provided useful resources to ease the research process. Although there seemed to be a lot of news and media articles on medicines, organisation and government coverage was limited. It was interesting to read about the different perspectives and reasons behind the price hikes, and also what countries aim to achieve by implementing certain costs to essential medicines. Despite the research process being demanding, the journey not only allowed me to, hopefully, enhance my research skills, but also learn a lot of medical information along the way. Overall I found my experience with the E$$ENTIAL MEDICINE$ challenges very engaging and rewarding. It gave me a better insight into the need to make essential medicines more accessible and affordable for everyone.

**Comment by amychau16** [comment]
Hi Shirley!

It was interesting to read your learning journey (especially how we had some overlaps in our chose medicines) and to compare my own in retrospect.

- I also researched insulin for one of my challenges. Although this is a very common medication and there was an abundance of information on the web, I felt that insulin was very difficult to research especially for price hikes. I felt that there was so much information scattered on price increases and pricing systems that I spent a lot of time synthesising everything I found across multiple sources so that I could enter my response. Despite this challenging and confusing process, this experience stood out the most for me as I not only built on my knowledge regarding pricing systems, but I also further improved my research skills. It seems that you had a completely different experience here, so I'm wondering if this has to do with our different research abilities or the different search terms we used?

> **I am interested to learn more about the different medicines which have been repurposed to treat illnesses. I found it especially interesting that drugs like fluoxetin, which are more older in their discovery, can still remain relevant today, even with modern technology and more advanced research.**

It was interesting to read that we had the exact same experience whilst researching about fluoxetine. Learning that a medicine is so relevant to our current context was definitely a motivating factor for me during the research process. However, after reading some of other members' journals, it is important for us to consider if this will be the same experience for future participants, especially if they may not already have existing interest in medicines...

> **I especially found it challenging to find reliable information on the price hikes of different drugs and there was a lot of varying statistics which meant filtering the different sources was important.**

- I also found that to be the most challenging aspect of completing the Price Hike$ challenge. Although it fuelled my curiosity to continue researching which helped with developing my research inquiry skills (in terms analysing and deciding which sources had reliable information), it was a frustrating and overwhelming experience. I think that this will definitely be something that future participants will encounter, so I think a guide on potential websites that usually have relevant information (e.g. NCBI) may be useful...
